### DISCUSSION TITLE
Dịch one-vs-all, one-vs-rest, one-vs-one

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#18] [Terminology] [answered]
Có nên dịch không nhỉ mọi người? Nếu có thì mình tạm đề xuất dịch là:
* `one-vs-all`: `một so với tất cả`
* `one-vs-rest`: `một so với phần còn lại`
* `one-vs-one`: `một so với một`

@mlbvn/handson-ml2

**Comment by quangvnai** [comment]
`chọi` nên đổi thành `với` hoặc giữ nguyên `vs`? Nghe `chọi` mang màu sắc đấu đá quá :v

**Comment by thanhcsf** [comment]
Hoặc là **so với** ....
Ps1: anh thấy nên dịch.
Ps2: tiêu đề nó không hiện backtick nên chắc thôi mình không cần đánh backtick đâu em ạ.

**Reply by tiepvupsu** [reply]
Anh lại thấy không nên dịch, mình để chú thích hoặc ngoặc đơn khi gặp lần đầu. Cụm tiếng Anh rất ngắn và dễ hiểu.

Nếu mọi người thực sự muốn dịch thì anh đề xuất:
* `một-toàn bộ`
* `một-một`
* `một-còn lại`

**Reply by minhduc0711** [reply]
@tiepvupsu em nghĩ nếu dịch được thì cứ dịch thôi ạ, tại mấy từ như `code` mình cũng dịch hết.
Em thấy đề xuất của anh ngắn gọn và ổn hơn ạ.

**Comment by minhduc0711** [comment, marked as answer]
tạm chốt nhé mọi người
     * `một-toàn bộ`
     * `một-một`
    * `một-còn lại`
